.gitignore

```diff
@@ -24,6 +24,9 @@ x64/
 # Enable "build/" folder in the NuGet Packages folder since NuGet packages use it for MSBuild targets
 !packages/*/build/
 
+# Debug artifacts
+launchSettings.json
```

Comment:
Is this change intentional?

Reply:
Yes. This stops Git from constantly including launchSettings.json whenever I setup Visual Studio for debugging.